### PR TITLE
Feature: Update application status #238

### PR DESCRIPTION
**Pull request by rachelchenrc**
[Trello card](https://trello.com/c/y34mZqUC/238-as-a-user-i-want-to-be-able-to-update-the-status-of-an-application)
[Live env](https://nj-career-network-dev4.firebaseapp.com/application-tracker/)

Change in data mode:

- Add isActive in each application
- Store currentStatus for convenience in each application
- Rename timestamp in application to be a descriptive:  lastUpdateTimestamp

### CODE REVIEW

File: components/applicationTracker/ApplicationUpdateDialog/ApplicationUpdateDialog.js [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+              label="This is an Active Application"
+              control={
+                <Checkbox
+                  checked={values.isActive || false}
+                  onChange={event => setValues({ ...values, isActive: event.target.checked })}
+                  color="primary"
+                />
+              }
+            />
+            {JSON.stringify(values)}
```
Comment (hannahtwobulls) [inline components/applicationTracker/ApplicationUpdateDialog/ApplicationUpdateDialog.js:155]: Probs remove this

**Comment by hannahtwobulls** [comment]
Besides the json in the dialog, it looks AWESOME

File: components/applicationTracker/ApplicationUpdateDialog/ApplicationUpdateDialog.js (hunk 1 above)
Comment (rachelchenrc) [reply components/applicationTracker/ApplicationUpdateDialog/ApplicationUpdateDialog.js]: removed!

**Comment by rachelchenrc** [comment]
Empty state for the list was added.

**Pull request merged**